server/controller/user/orderRequest.js

```diff
@@ -0,0 +1,25 @@
+const { addOrderRequest } = require('../../database/queries');
+
+const userOrderReguest = async (req, res, next) => {
+  try {
+    const {
+      body: { providerId, description },
+      user: { id },
+    } = req;
+
+    await addOrderRequest({
+      providerId,
+      description,
+      id,
+    });
+
+    res.status(200).json({
+      status: 200,
+      message: 'Order request sent successfully',
+    });
```

Comment:
the status Code here is [201](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201)